.circleci/config.yml

```diff
@@ -130,6 +130,35 @@ jobs:
             - ~/.ccache
             - ~/.cache/pip
 
+  arm64:
```

Comment:
Maybe we should call it `linux-arm64` because we might want to also test for `macos-arm64` which is quite a different beast in the future if some CI provider start to provide Apple Silicon machines.